packages/runtime-common/prettier-config.ts

```diff
@@ -0,0 +1,211 @@
+// Configuration Resolution Design
+// This module provides configuration resolution for prettier integration
+
+import type { Options as PrettierOptions } from 'prettier';
+
+export interface PrettierConfig extends PrettierOptions {
+  parser?: string;
+  plugins?: string[];
+  singleQuote?: boolean;
+  printWidth?: number;
+  tabWidth?: number;
+  useTabs?: boolean;
+  semi?: boolean;
+  [key: string]: any;
+}
+
+export interface FileTypeInfo {
+  type: string;
+  parser: string;
+  confidence: 'high' | 'medium' | 'low';
```

Comment:
What is this used for? I don’t see it consumed anywhere

Reply:
just used locally